src/librbml/lib.rs

```diff
@@ -772,11 +924,29 @@ pub mod writer {
             let cur_pos = try!(self.writer.tell());
             try!(self.writer.seek(last_size_pos as i64, old_io::SeekSet));
             let size = cur_pos as uint - last_size_pos - 4;
-            try!(write_sized_vuint(self.writer, size, 4));
-            let r = try!(self.writer.seek(cur_pos as i64, old_io::SeekSet));
+
+            // relax the size encoding for small tags (bigger tags are costly to move).
```

Comment:
"relaxation" is an established technical term for this from the assembler world. See [assembler releaxation](http://eli.thegreenplace.net/2013/01/03/assembler-relaxation).